packages/core/src/update/update.ts

```diff
@@ -0,0 +1,93 @@
+import fs from "fs";
+import path from "path";
+import spawn from "cross-spawn";
+
+export function run(rootDir: string) {
+  const manager = fs.existsSync(path.join(rootDir, "yarn.lock")) ? Yarn : NPM;
+  const json = JSON.parse(
+    fs.readFileSync(path.join(rootDir, "package.json")).toString()
+  );
+
+  manager.update({
+    root: rootDir,
+    type: json.dependencies["@serverless-stack/cli"]
+      ? "dependencies"
+      : "devDependencies",
+    pkgs: ["@serverless-stack/cli@latest"],
+  });
+  manager.update({
+    root: rootDir,
+    type: json.dependencies["@serverless-stack/resources"]
+      ? "dependencies"
+      : "devDependencies",
+    pkgs: ["@serverless-stack/resources@latest"],
+  });
+
+  const compare = JSON.parse(
+    fs
+      .readFileSync(
+        path.join(
+          rootDir,
+          "node_modules/@serverless-stack/resources/package.json"
+        )
+      )
+      .toString()
+  );
+
+  function cdk(type: "dependencies" | "devDependencies") {
+    const updates = Object.keys(json[type] || {})
+      .filter((key) => key.startsWith("@aws-cdk"))
+      .filter((key) => json.dependencies[key] || json.devDependencies[key])
+      .filter((key) => compare.dependencies[key])
```

Comment:
Does this mean we are only updating the @aws-cdk packages that exist in resources? (We should update all @aws-cdk packages, CDK doesn't handle mixed package versions very well)

Reply:
Yeah good point.  I updated to use whatever `@serverless-stack/core` is using